contracts/apps/disputable/IAgreement.sol

```diff
@@ -16,12 +16,14 @@ contract IAgreement is IArbitrable, IACLOracle {
     event ActionClosed(uint256 indexed actionId);
     event ActionChallenged(uint256 indexed actionId, uint256 indexed challengeId);
     event ActionSettled(uint256 indexed actionId, uint256 indexed challengeId);
-    event ActionDisputed(uint256 indexed actionId, uint256 indexed challengeId);
+    event ActionDisputed(uint256 indexed actionId, uint256 indexed challengeId, uint256 indexed disputeId);
```

Comment:
Discussed offline, we agreed on removing the `disputeId` since:
1. The information can be retrieved from the exposed getters
2. AragonCourt is already emitting an event indexing the address of the subject (agreement in this case)